Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -75,284 +76,169 @@ public class ProjectController : Controller
+      *   "IsActive":"True",
+      *   "TeamLeaderId":"1",
+      *   "ApplicationUsers":"List of Users"
+      * }      
       * @apiSuccessExample {json} Success-Response:
       * HTTP/1.1 200 OK 
       * {
-      *     "description":"Add Project in ProjectTable"
+      *     "description":"Add new project"
       * }
       */
         [Authorize]
         [HttpPost]
-        [Route("addProject")]
-        public async Task<IActionResult> addProject([FromBody]ProjectAc project)
+        [Route("")]
+        public async Task<IActionResult> AddProject([FromBody]ProjectAc project)
         {
-            try
+            var createdBy = _userManager.GetUserId(User);
+            if (ModelState.IsValid)
             {
-                var createdBy = _userManager.GetUserId(User);
-                if (ModelState.IsValid)
+                ProjectAc projectAc =await _projectRepository.CheckDuplicateProject(project);
+                
+                if (!string.IsNullOrEmpty(projectAc.Name) && !string.IsNullOrEmpty(projectAc.SlackChannelName))
                 {
-                    ProjectAc p = _projectRepository.checkDuplicate(project);
-                    if (p.Name != null && p.SlackChannelName != null)
+                    int projectId = await _projectRepository.AddProject(project, createdBy);
+                    foreach (var applicationUser in project.ApplicationUsers)
                     {
-                        int id = await _projectRepository.AddProject(project, createdBy);
-                        foreach (var applicationUser in project.ApplicationUsers)
-                        {
-                            ProjectUser projectUser = new ProjectUser();
-                            projectUser.ProjectId = id;
-                            projectUser.UserId = applicationUser.Id;
-                            projectUser.CreatedBy = createdBy;
-                            projectUser.CreatedDateTime = DateTime.UtcNow;
-                            _projectRepository.AddUserProject(projectUser);
-                        }
-                        return Ok(project);
+                        ProjectUser projectUser = new ProjectUser();
+                        projectUser.ProjectId = projectId;
+                        projectUser.UserId = applicationUser.Id;
+                        projectUser.CreatedBy = createdBy;
+                        projectUser.CreatedDateTime = DateTime.UtcNow;
+                        await _projectRepository.AddUserProject(projectUser);
                     }
-                    else
-                    { return Ok(project); }
+                    return Ok(project);
                 }
-                return Ok(false);
-            }
-            catch (Exception ex)
-            {
-                ex.ToExceptionless().Submit();
-                throw ex;
+                else
+                { return Ok(project); }
             }
+            return Ok(false);
```

Comment:
if project name or project slack channel name is already exists I have return project object where project name or project  slack channel name is null using this I have display the proper message to end user whether project name or project slack channel name is already exits. but now i have updated my return method instead of return Ok(false) I have return badrequest() if modelstate is not valid.